<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTableTime</class>
 <widget class="QWidget" name="FormTableTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="dateTimeEditFrom">
     <property name="displayFormat">
      <string>d MMM yyyy hh:mm</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>to</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="dateTimeEditTo">
     <property name="displayFormat">
      <string>d MMM yyyy hh:mm</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Time zone</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxTimezone"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
